Tests/Unit/Controller/TeaControllerTest.php

```diff
@@ -40,9 +40,13 @@ protected function setUp(): void
         parent::setUp();
 
         // We need to create an accessible mock in order to be able to set the protected `view`.
+        $methodsToMock = ['redirectToUri', 'htmlResponse'];
+        if (method_exists(TeaController::class, 'forward')) {
```

Comment:
I'd prefer to use a TYPO version switch for this. This will make removing the corresponding code parts easier when we drop support for V11:

```php
if ((new Typo3Version())->getMajorVersion() <= 11) {
```